.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+tab_width = 4
```

Comment:
[https://www.python.org/dev/peps/pep-0008/#indentation](https://www.python.org/dev/peps/pep-0008/#indentation)

This is not how python understand (\t) is how a text editor understand a (\t).

Puting that a tab (in case of use) is going to be the equivalent of 8 spaces breaks rules of PEP8 and also will break the PEP8 tests in ckan